platform/darwin/src/string_nsstring.mm

```diff
@@ -27,5 +27,24 @@
     return result;
 }
 
+std::string formatNumber(double number,
+                         const std::string& localeId,
+                         const std::string& currency,
+                         uint8_t minFractionDigits,
+                         uint8_t maxFractionDigits) {
+    NSNumberFormatter *_numberFormatter;
+    _numberFormatter = [[NSNumberFormatter alloc] init];
+    _numberFormatter.locale = !localeId.empty() ? [NSLocale localeWithLocaleIdentifier:@(localeId.c_str())] : nil;
```

Comment:
We should make sure the default locale matches the default locale of the comparison expression operators as well as the number formatter in GL JS. `nil` results in either the application’s current locale or the system preferred locale; can’t remember which off the top of my head.

Reply:
I checked the NSNumberFormatter documentation, but couldn't find what's the default locale in case of `nil`.